agent/dockerclient/dockerapi/docker_client_test.go

```diff
@@ -1008,6 +1008,34 @@ func TestDockerVersion(t *testing.T) {
 	assert.Equal(t, "1.6.0", str, "Got unexpected version string: "+str)
 }
 
+func TestSystemPing(t *testing.T) {
```

Comment:
this and below could be combined to table driven tests

Reply:
For these two I will prefer to keep separate because I will have to add custom logic anyway for each test for asserting error and nil response.